gmail/src/services/log_email.ts

```diff
@@ -1,17 +1,35 @@
 import { postJsonRpc } from "../utils/http";
 import { URLS } from "../const";
 import { State } from "../models/state";
+import { Email } from "../models/email";
 
 /**
  * Log the given email body in the chatter of the given record.
  */
-export function logEmail(recordId: number, recordModel: string, emailBody: string): boolean {
+export function logEmail(recordId: number, recordModel: string, email: Email): boolean {
     const url = State.odooServerUrl + URLS.LOG_EMAIL;
     const accessToken = State.accessToken;
 
+    const attachments = [];
+
+    // The size limit of the POST request are 50 MB
+    // So we limit the total attachment size to 40 MB
+    const MAXIMUM_ATTACHMENTS_SIZE = 40 * 1024 * 1024;
+
+    let totalAttachmentsSize = 0;
+
+    for (const attachment of email.getAttachments()) {
+        totalAttachmentsSize += attachment.bytesSize;
+        if (totalAttachmentsSize < MAXIMUM_ATTACHMENTS_SIZE) {
+            attachments.push([attachment.name, attachment.content]);
+        } else {
+            Logger.log(`The file ${attachment.name} is ignored because you overcome the total attachment size limit.`);
```

Comment:
"overcome" is usually used in another context (when you "overcome" an obstacle or something).
Here "exceed" is generally better / more technical.

(Same thing goes for your comme message, I think I saw a typo "if" -> "is" in there as well)